browser/css/notebookbar.css

```diff
@@ -379,12 +379,15 @@ button.ui-tab.notebookbar {
 }
 
 #fontsizecombobox.notebookbar {
-	width: 50px !important;
+	width: 5em;
 }
 
 .notebookbar.ui-combobox * {
 	line-height: 22px;
 }
+.ui-combobox-content {
+	padding-left: 5px;
+}
```

Comment:
Thanks we really needed padding here. I wonder where did you remove it. It seems it was not in f7491e1f62de5401529db40b8b899e5c0563badc :thinking:

Reply:
Could change it to `padding-inline` so both we had padding on both sides and so languages such as `&lang=ar` (LTR) also get the fix? And maybe increase the padding to `8px`

Reply:
Either way it should be added on both sides. Thus my suggestion for padding-inline

Reply:
> Thus my suggestion for padding-inline

I have done that.